services/state/roster/roster.proto

```diff
@@ -107,4 +104,5 @@ message RosterEntry {
      * This list SHALL NOT be empty.<br/>
      */
     repeated proto.ServiceEndpoint gossip_endpoint = 5;
+
 }
```

Comment:
Not a blank line before the end curly brace, we require a newline at EOF (github adds a warning symbol to show where that's missing).
The consistent presence of the newline helps some of the tooling to handle all files consistently.

Reply:
sorry about that. I just checked and didn't get the newline warning in intellij. Guessing i have to enable it, i'll look for it. Updated now